drivers/adc/ad7124/ad7124.c

```diff
@@ -403,6 +439,27 @@ int32_t ad7124_read_data(struct ad7124_dev *dev,
 	return ret;
 }
 
+/**
+ * @brief Get the ID of the channel of the latest conversion.
+ *
+ * @param dev     - The handler of the instance of the driver.
+ * @param status  - Pointer to store the read data.
+ *
+ * @return Returns 0 for success or negative error code.
+ */
+int32_t ad7124_get_read_chan_id(struct ad7124_dev *dev, uint32_t *status)
```

Comment:
Maybe move function this function is separate commit since it is out of the commit's subject?

It is not quite a wrapper, rather a function that uses wrappers and adds new functionality to the driver.

Reply:
Makes sense. Will do.